shared/xccdf/services/base.xml

```diff
@@ -117,6 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
+<platform idref="cpe:/o:virtualization:baremetal"/>
```

Comment:
Clarification: What's the difference between a Wind River Linux (OVP) baseline running on bare metal and a RHEL baseline running on bare metal ?  Do both of them need to be indicated with "cpe:/o:virtualization:baremetal" ?  I would expect the CPE to be something more like "cpe:/o:windriver:wrlinux:9:ovp:baremetal" or "cpe:/o:redhat:enterprise_linux:7:virtualization:baremetal".